app/controllers/budgets_controller.rb

```diff
@@ -6,8 +6,12 @@ def index
   end
 
   def new
-    @group = Group.find(params[:group_id])
     @budget = Budget.new
+    @group = Group.find(params[:group_id])
+
+    return unless @group.nil?
+
+    @group = []
```

Comment:
- [ ] You did a good job working on the changes suggested by the previous reviewer. However, I'm having some problems trying to create a new "Budget":

![image](https://github.com/josuerushanika/budget-app/assets/77069011/79f7783b-075d-4960-b236-ef74bbef700e)

If you see, after the POST starts, a `RoutingError` happens. This is because you didn't create a route for your `Budget#create` action. Please consider adding it! 🤓 🔧 


- [ ] **[OPT]** Also, as a suggestion, for user experience:

![image](https://github.com/josuerushanika/budget-app/assets/77069011/93078561-4817-46ed-9c61-97fa04336f6e)

It would be great if you remove the `Category` field from your `Budget` form. This is because you're using the `group_id` from the params which is a good approach. Keeping this input generates this warning:

![image](https://github.com/josuerushanika/budget-app/assets/77069011/ab3f8525-3752-4095-a7c0-2bbd42ff827f)

Consider fixing it! 🤓 🔧